db2/cache.go

```diff
@@ -0,0 +1,118 @@
+package db2
+
+import (
+	"encoding/json"
+	"sync"
+	"time"
+)
+
+// var ttl = 20_000 * time.Millisecond
+var oneBlockTTL = 1 * time.Second
```

Comment:
We can define oneBlockTTL and blocksTTL as const values since they are not modified anywhere in the code